js/githubresume.js

```diff
@@ -7,7 +7,7 @@ var trackerId = 'UA-21222559-1';
         a = /\+/g,  // Regex for replacing addition symbol with a space
         r = /([^&=]+)=?([^&]*)/g,
         d = function (s) { return decodeURIComponent(s.replace(a, " ")); },
-        q = window.location.search.substring(1);
+        q = window.location.search.substring(1).split("/")[0];
```

Comment:
I don't think this is a bug.Like this `https://github.com/search?utf8=%E2%9C%93&q=user%3Adhh` and `https://github.com/search?utf8=%E2%9C%93&q=user%3Adhh/`.
`istx25/` is a valid name as a parameter.